internal/docs/exported_fields.go

```diff
@@ -46,13 +54,37 @@ func renderExportedFields(fieldsParentDir string) (string, error) {
 		builder.WriteString("(no fields available)\n")
 		return builder.String(), nil
 	}
-	renderFieldsTable(&builder, collected)
+
+	packageRoot, found, err := packages.FindPackageRoot()
+	if err != nil {
+		return "", fmt.Errorf("locating package root failed: %w", err)
+	}
+	if !found {
+		return "", errors.New("package root not found, you can only create new data stream in the package context")
+	}
+	manifest, err := packages.ReadPackageManifestFromPackageRoot(packageRoot)
+	if err != nil {
+		return "", fmt.Errorf("failed to read package manifest from \"%s\"", packageRoot)
+	}
+	sv, err := semver.NewVersion(manifest.SpecVersion)
+	if err != nil {
+		return "", fmt.Errorf("failed to obtain spec version from package manifest in \"%s\"", packageRoot)
+	}
+
+	if sv.LessThan(semver3_2_3) {
+		renderFieldsTable(&builder, collected, false, false)
+	} else {
+		renderFieldsTable(&builder, collected, true, true)
+	}
```

Comment:
Nit. I would suggest something like this for self-explanatory purposes:
```suggestion
	renderExamples := true
	renderValues := true
	if sv.LessThan(semver3_2_3) {
		renderExamples = false
		renderValues = false
	}
	renderFieldsTable(&builder, collected, renderValues, renderExamples)
```

Reply:
Implemented suggestion